auraed/tests/common/cells.rs

```diff
@@ -82,12 +79,12 @@ impl ExecutableBuilder {
     pub fn new() -> Self {
         Self {
             name: format!("ae-sleeper-{}", uuid::Uuid::new_v4()),
-            command: "sleep 400".to_string(),
+            command: "sleep 5".to_string(),
```

Comment:
The reason why I prefer a longer timeout is so I can poke around in `/proc` if I'm trying to figure something out.